QEfficient/utils/device_utils.py

```diff
@@ -27,3 +28,41 @@ def get_available_device_id():
             elif "Failed to find requested device ID" in result.stdout:
                 print("Failed to find requested device ID")
                 return None
+            
+def is_qpc_size_gt_32gb(params: int, mxfp6: bool) -> bool:
+    if mxfp6:
+        qpc_size = math.ceil((params * 1) / Constants.GB)
+    else:
+        qpc_size = math.ceil((params * 2) / Constants.GB)
+
+    print(f"Approximate QPC size is: {qpc_size} GB")
+    print("Number of Devices required: ", math.ceil(qpc_size / Constants.MAX_QPC_LIMIT))
+    return qpc_size > Constants.MAX_QPC_LIMIT
+
+
+def is_multi_qranium_setup_available():
+    result = None
+    command = ["/opt/qti-aic/tools/qaic-util", "-q"]
+    try:
+        result = subprocess.run(command, stdout=subprocess.PIPE, universal_newlines=True)
+        filtered_result = subprocess.run(
+            ["grep", "Device Capabilities"], input=result.stdout, stdout=subprocess.PIPE, text=True
+        )
+    except OSError:
+        print("Command not found", command)
+        return None

```

Comment:
Should this return `False` here?

Reply:
No, it raises error if exception is thrown and None is returned, similar to `get_available_device_id` func